src/Entity/Consumer.php

```diff
@@ -23,39 +24,45 @@ public function __construct()
     {
         $this->reviews = new ArrayCollection();
     }
-
-    public function __toString()
-    {
-        return $this->consumer_name;
-    }
-
+//    public function __toString()
+//    {
+//        return $this->consumer_name;
+//    }
     /**
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
      */
     private $id;
-
     /**
-     * @ORM\Column(type="string", length=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
      */
     private $consumer_name;
-
     /**
-     * @ORM\Column(type="string", length=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
      */
     private $consumer_lastname;
-
     /**
      * @ORM\Column(type="string", length=255)
      */
     private $consumer_email;
-
     /**
-     * @ORM\Column(type="string", length=255)
+     * @ORM\Column(type="string", length=255, nullable=true)
      */
     private $consumer_password;
 
+    private $yes;
```

Comment:
Just a suggestion to use more semantical variable names, such as isLiked. Would be difficult to figure out what this means after couple of months (got the same suggestion from Paulius about my variable names) :)

Reply:
ok